### DISCUSSION TITLE
Set tile count

### DISCUSSION THREAD

**Discussion by AaronAnz** [dlemstra/Magick.NET#955] [Help] [answered]
How can I force all montage images that are created to max out at 5 tiles horizontally?

**Comment by dlemstra** [comment, marked as answer]
Setting the `TileGeometry` of the `MontageSettings` to `new MagickGeometry(5,0)` should do that.

**Reply by AaronAnz** [reply]
Apparently I'm stupid, I swear I had already tried that. Thank you!
